wsaa.py

```diff
@@ -136,42 +138,62 @@ def sign_tra(tra, cert=CERT, privatekey=PRIVATEKEY, passphrase=""):
             cert = open(cert).read()
             if isinstance(cert, str):
                 cert = cert.encode("utf-8")
-        cert = x509.load_pem_x509_certificate(cert, default_backend())
+        cert = x509.load_pem_x509_certificate(cert)
 
-        try:
-            # Firmar el texto (tra) usando cryptography (openssl bindings para python)
-            p7 = _lib.PKCS7_sign(
-                cert._x509, private_key._evp_pkey, _ffi.NULL, bio_in, 0
-            )
-        finally:
-            # Liberar memoria asignada
-            _lib.BIO_free(bio_in)
-        # Se crea un buffer nuevo porque la firma lo consume
-        bio_in = _lib.BIO_new_mem_buf(tra, len(tra))
-        try:
-            # Crear buffer de salida
-            bio_out = _lib.BIO_new(_lib.BIO_s_mem())
+        if sys.version.split(" ")[0][0:3] == "2.7":
```

Comment:
Maybe you would only check the major version number.
Something like:
```suggestion
        MAJOR = 0 # first string element
        if sys.version[MAJOR] == "2":
```
or use the `version_info` method
```suggestion
        if sys.version_info.major == 2:
```